package.json

```diff
@@ -65,6 +65,7 @@
     "@angular/platform-browser-dynamic": "^5.0.0-rc.1",
     "@angular/platform-server": "^5.0.0-rc.1",
     "@angular/router": "^5.0.0-rc.1",
+    "@types/sprintf": "^0.1.0",
```

Comment:
What does it do for us?

Reply:
It is small library with function sprintf. It replace flags in string e.g %s with value from arguments list.

Reply:
Do we need additional dependency for this? I'd like at least to *try avoiding* adding too much dependencies...